app/views/admin/enterprises/form/_primary_details.html.haml

```diff
@@ -63,6 +63,13 @@
       %span{ ng: { class: 'availability.toLowerCase()', hide: "checking" } }
         {{ availability }}
         %i{ ng: { class: "{'icon-ok-sign': availability == 'Available', 'icon-remove-sign': availability == 'Unavailable'}" } }
+  .row{ ng: { show: "Enterprise.sells == 'own' || Enterprise.sells == 'any'" } }
+    .three.columns.alpha
+      = f.label :permalink, t('.ofn_uid')
+      %div{'ofn-with-tip' => t('.ofn_uid_tip', link: spree.root_url)}
+        %a= t('admin.whats_this')
+    .six.columns
+      = f.text_field :id, { 'ng-model' => "Enterprise.id", 'ng-model-options' => "{ updateOn: 'default blur', debounce: {'default': 300, 'blur': 0} }", 'ng-disabled' => true }
```

Comment:
this is wrong, you dont want a text field, you only want to print the id
maybe just
= {{Enterprise.id}}

Reply:
Right, I will remove all this Angular stack.